cmd/livepeer/starter/starter.go

```diff
@@ -294,6 +296,7 @@ func DefaultLivepeerConfig() LivepeerConfig {
 		MaxTotalEV:              &defaultMaxTotalEV,
 		DepositMultiplier:       &defaultDepositMultiplier,
 		MaxPricePerUnit:         &defaultMaxPricePerUnit,
+		IgnoreMaxPriceIfNeeded:  &defaultEnforceMaxPrice,
```

Comment:
@leszko looks good to me thanks! However is there a reason the default parameter name was not refactored while the test was?